<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectoryPairChooserWidget</class>
 <widget class="QWidget" name="DirectoryPairChooserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>82</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelDevice">
     <property name="text">
      <string>Encrypted data location</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="editDevice">
     <property name="mode">
      <set>KFile::Directory|KFile::File|KFile::LocalOnly</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelMountPoint">
     <property name="text">
      <string>Mount point</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="editMountPoint">
     <property name="mode">
      <set>KFile::Directory|KFile::File</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
